### PR TITLE
#168857238 chore(ui): fix LFA feedbacks

### PR DESCRIPTION
**Pull request by bihire**
### What does this PR do ?
Implement changes from LFA feedbacks
### Description of tasks to be completed ?
Make the following changes in the ui
- change the page main color
- change buttons look in flag page
- change add article button to deep blue
- reduce top margin from the main content
### How should this be manually tested ?
After cloning this repo, od into it and open the following pages
- index.html
- article.html
- flaggedArticle.html
- flaggedComment.html
- admin_dashboard.html
### Any background context you want to provide ?
N/A
### What are the relevant PivotTracker stories ?
#168857238

image
Annotation 2019-10-01 010349

Annotation 2019-09-30 234847

### CODE REVIEW

**Comment by NkFab** [comment]
Good work.

**Review by NkFab — APPROVED** [review]

**Review by NkFab — APPROVED** [review]

File: UI/css/welcome.css [hunk 1]
```diff
@@ -104,16 +105,17 @@ input#email,#username{
       font-size: 18px;
       /* border-radius: 25px; */
```
Comment (NkFab) [inline UI/css/welcome.css]: Remove commented lines.

File: UI/css/welcome.css [hunk 2]
```diff
@@ -104,16 +105,17 @@ input#email,#username{
...
       box-shadow:none;
-      background: #F20732;
+      /* background: #F20732; */
+      background: #008EFA;
       color: #fff;
 }
 .card-item-content:hover {
     background: #fff;
-    border: 1px solid #F20732;
+    /* border: 1px solid #008EFA; */
```
Comment (NkFab) [inline UI/css/welcome.css]: Remove commented lines.

File: UI/css/welcome.css [hunk 3]
```diff
@@ -184,7 +194,8 @@ span {
       /* border-radius: 25px; */
```
Comment (NkFab) [inline UI/css/welcome.css]: Remove commented lines.

File: UI/css/welcome.css [hunk 4]
```diff
@@ -184,7 +194,8 @@ span {
...
       box-shadow:none;
       
-      background: #F20732;
+      background: #007FE0;
+      /* background: #F20732; */
```
Comment (NkFab) [inline UI/css/welcome.css]: Remove commented lines.

**Review by NkFab — COMMENTED** [review]
Remove all the commented lines that do not describe what a statement is doing.

File: UI/css/welcome.css (hunk 4 above)
Comment (bihire) [reply UI/css/welcome.css]: ok, done

**Pull request merged**